server/schemas/video.schema.js

```diff
@@ -0,0 +1,20 @@
+import Joi from '@hapi/joi';
+import {
+  requiredObjectId,
+  requiredString,
+  optionalString,
+  optionalObjectId,
+} from './helper.schema';
+
+export const addVideoSchema = Joi.object().keys({
+  title: requiredString('Video Name'),
+  url: requiredString('Video URL'),
+  propertyId: requiredObjectId('Property id'),
+});
+
+export const updateVideoSchema = Joi.object({
+  videoId: requiredObjectId('Video id'),
```

Comment:
Let's make this one `id`

```suggestion
  id: requiredObjectId('Video id'),
```